### PR TITLE
Fix potential medium/high severity security holes

### PR DESCRIPTION
**Pull request by systemshift**
In order to # #3036 to merge on master, some minor edits need to be made and remove the few warnings that show up.

### CODE REVIEW

**Comment by karlhigley** [comment]
It looks like `literal_eval` is causing a bunch of test failures and hanging the build. 😢

**Comment by karlhigley** [comment]
If that turns out to be difficult to fix, I think you can label the line with `#nosec` to exclude it from the scan.

**Comment by karlhigley** [comment]
I think the reason `literal_eval` doesn't work here is that `Plan` is actually relying on `eval` to execute commands, not just convert strings to literal data structures. The commands that get evaluated by the `Plan` tests are:

- 'torch.nn.functional.linear'
- 'torch.nn.functional.relu'
- 'torch.nn.functional.log_softmax'

I made a [change](https://github.com/OpenMined/PySyft/pull/2951/files) recently that addressed another `eval`, and maybe we could take a similar approach here.

**Comment by karlhigley** [comment]
Since commands could also be executed on other frameworks (i.e. not PyTorch) though, maybe the best thing to do for now is just label this line so the security checks ignore it.
